package.json

```diff
@@ -20,8 +20,8 @@
             "path": "Samples~/Cinemachine Example Scenes"
         },
         {
-            "displayName": "Scrub Bubble support for Nested Timelines (experimental)",
-            "description": "Experimental feature.  Adds a Timeline extension to support Scrubb Bubble for nested timelines.  This extension will become obsolete when Timeline adds native support for the missing API",
+            "displayName": "Cached Scrubbing support for Nested Timelines (experimental)",
+            "description": "Experimental feature.  Adds a Timeline extension to support Cached Scrubbing for nested timelines.  This extension will become obsolete when Timeline adds native support for the missing API",
             "path": "Samples~/Nested Timeline Scrub Bubble"
```

Comment:
Did a search for Scrub Bubble, and found in CinemachineTimelineEditorExtensions.cs. 
And the name of the samples is still scrub bubble here.

Reply:
It's not a dirty word... I think we can leave it